app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -48,30 +48,30 @@
       Resources
     </h3>
     <div id="mod-center-other-options">
-      <% if @community_mod_channel %>
-        <a href="/connect/<%= @community_mod_channel.slug %> " target=" _blank " rel="noopener noreferrer nofollow" class="crayons-link crayons-link--block mod-center--nav__item">
-          Connect Channel
-          <%= inline_svg_tag("link-external.svg", aria: true, title: "External link", class: "crayons-icon") %>
-        </a>
-      <% end %>
       <a href="<%= URL.url %>/community-moderation" class="crayons-link crayons-link--block mod-center--nav__item">
-        Moderator Guide
+        Trusted User Guide
+        <%= inline_svg_tag("link-external.svg", aria: true, title: "External link", class: "crayons-icon") %>
+      </a>
+      <a href="<%= URL.url %>/tag-moderation" class="crayons-link crayons-link--block mod-center--nav__item">
+        Tag Moderation Guide
         <%= inline_svg_tag("link-external.svg", aria: true, title: "External link", class: "crayons-icon") %>
       </a>
       <a href="<%= URL.url %>/terms" class="crayons-link crayons-link--block mod-center--nav__item">
-        Terms Of Conduct
+        Terms & Conditions
```

Comment:
"Terms & Conditions" should be changed to "Terms of Use"

(Sorry, not sure how to seamlessly "propose a change")